src/clj/fluree/db/json_ld/policy/rules.cljc

```diff
@@ -140,50 +140,31 @@
    {}
    policy-rules))
 
-;; TODO - For now, extracting a policy from a `select` clause does not retain the
-;  @value: 'x', @type: '@json' structure for the value of `f:query` which
-;  then creates an issue with JSON-LD parsing. This adds back the
-;  explicit @type declaration for the query itself. Once there is a way
-;  to have the query result come back as raw json-ld, then this step can
-;  be removed.
-(defn policy-from-query
-  "Recasts @type: @json from a raw query result which
-  would looses the @type information."
-  [query-results]
-  (mapv
-   #(if-let [query (get % const/iri-query)]
-      (assoc % const/iri-query {"@value" query
-                                "@type"  "@json"})
-      %)
-   query-results))
+(defn validate-values-map
+  [values-map]
+  (or (map? values-map)
+      (throw (ex-info (str "Invalid policy values map. Must be a map. Received: " values-map)
+                      {:status 400
+                       :error  :db/invalid-values-map}))))
+
+(defn expanded?
```

Comment:
This is fine for now, but I would prefer more robust code paths where we only call this function with data we already know is expanded. That has its own benefits, and this method introduces a dependency on the proprietary `:idx` key and limits our flexibility with respect to expansion going forward. Making the code paths more robust is probably a much larger refactor, so I'm ok with this for now.

Reply:
Good point. I ran into this issue when using with fluree/server so added a quick fix.

I just fixed it up property and removed this.